<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/phieunopphat/KTLT(7).png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblhoten">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Họ &amp; Tên:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblsdt">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>SĐT:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbltensach">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Tên sách:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblmasach">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Mã sách:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblngaytradk">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Ngày trả dự kiến:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblngaytratt">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Ngày trả thực tế:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblngaymuon">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Ngày mượn:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblsongayquahan">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Số ngày quá hạn:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblsotiencantra">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Số tiền cần trả:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbltienphuthu">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Cửa hàng phụ thu 10.000 VNĐ trên mỗi ngày trả quá hạn </string>
    </property>
   </widget>
   <widget class="QPushButton" name="ptnThanhtoan">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(255, 255, 255);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string>THANH TOÁN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledHoten">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledSdt">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledTensach">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledMasach">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledNgaymuon">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledNgaytratt">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledSotiencantra">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledNgaytradk">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledSongayquahan">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pic4.qrc"/>
 </resources>
 <connections/>
</ui>
